contracts/NetworkSettings.sol

```diff
@@ -91,12 +76,14 @@ contract NetworkSettings is INetworkSettings, Owned, Utils {
 
     /**
      * @dev sets the network fee
-     * can be executed only by the owner
      *
-     * @param newNetworkFee new network fee in ppm units
+     * Requirements:
```

Comment:
we need to remember to point out that these are in PPM (and later on rename it to feePPM)

Reply:
I'll add a comment

Reply:
cool